domain/src/Access/DomainAccessCheck.php

```diff
@@ -10,7 +10,7 @@
 use Drupal\Core\Access\AccessCheckInterface;
 use Drupal\Core\Access\AccessResult;
 use Drupal\Core\Session\AccountInterface;
-use Drupal\domain\DomainInterface;
+use Drupal\domain\Entity\Domain;
```

Comment:
Good point. Tweaked as it was for type hinting in the `access()` method.
Somehow missed the fact that `getActiveDomain()` does return a DomainInterface